File: Programacion Orinetada a Ojectos/javascript/Car.js
function Car (licene,driver) {
    this.id;
    this.licene = licene;
    this.driver = driver;
    this.passenger;
}

Car.prototype.imprime= function(){
    console.log(this.driver)
    console.log(this.driver.name)
    console.log(this.driver.document)
}

File: Programacion Orinetada a Ojectos/Car.java
public class Car {
    Integer id;
    String license;
    Account driver;
    Integer passengenger;

    public Car(String license,Account driver) {
        this.license=license;
        this.driver=driver;
        
    }

    public void imprima() {
        System.out.println("metodo y el nombre es"+driver.name);
        
    }
    
}
